src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -62,6 +62,7 @@ mutable struct SEDistInfo
     sigma2::Union{Float64, Nothing}
     omega::Union{Float64, Nothing}
     eta::Float64
+    scaling::Float64
```

Comment:
This field is not explained in the docstring.

Reply:
I have added this field. It a parameter to ensure that in expectation the sketched norm-squared of the residual is the same as the norm squared of the residual.